packages/bpk-component-dialog/src/BpkDialog.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import BpkModal from 'bpk-component-modal';
+
+const BpkDialog = props => {
+  const { isOpen, onClose, target, dismissible, ...rest } = props;
+
+  return (
+    <BpkModal
+      isOpen={isOpen}
```

Comment:
technically you don't need to pipe these props through, `{...rest}` can take care of them, all you really need is:
```
return (
   <BpkModal
     {...rest}
     closeOnScrimClick={dismissible}
     showHeader={dismissible}
     fullScreenOnMobile={false}
   />
 );
```

Reply:
(We still need to specify the prop types mind you.)